create.py

```diff
@@ -3,43 +3,79 @@
 import warnings
 
 import pymysql as mdb
+import sqlite3
 
 from tt import *
 
 warnings.filterwarnings('ignore')
 
 
+def create_table_command_mysql():
+    sql_commands = [('''
+    use {db};'.format(args_dict['db']);
+    create table if not exists {table} (
+        id int not null auto_increment,
+        date varchar(10),
+        start time(0),
+        end time(0),
+        project varchar(8),
+        primary key (id)
+    );
+    ;
+    '''.format(db=args_dict['db'], table=args_dict['table']))]
+
+    sql_commands += ['create database if not exists {}'.format(args_dict['db'])]
+    return sql_commands
+
+def create_table_command_sqllite():
+    sql_commands = [('''
+    create table if not exists {table} (
+        id integer primary key autoincrement,
+        date varchar(10),
+        start time(0),
+        end time(0),
+        project varchar(8)
+    );
+    '''.format(table=args_dict['table']))]
+
+    return sql_commands
+
```

Comment:
I'd suggest handling that in a separate pull request, one that also defines a function to make the db connection based on the value of `dbengine`. Otherwise there will still be the need for the `if else` to decide to use `mdb.connect` or `sqlite3.connect` before calling `create_table_command`.